### PR TITLE
[jsk_baxter_startup] update baxter.launch to support current baxter-jsk-softhand

### PR DESCRIPTION
**Pull request by knorth55**
this PR is originally from [knorth55/eus_vive](https://github.com/knorth55/eus_vive/blob/master/launch/baxter/baxter.launch)
I'm moving the code from `eus_vive` to `jsk_robot`.

this PR updated `baxter.launch` to support current baxter-jsk-softhand.

this PR supports:
- New grippers 
  - softhand
  - softhand-v2
- New cameras
  - Torso realsense
  - Kodak Spherical camera
  - Spherical stereo camera
- New microphone
  - Torso respeaker
- Launch xdisplay

You can run this launch with following commands.

### For Baxter + Default gripper

```bash
roslaunch jsk_baxter_startup baxter_default.launch
```

### For Baxter + Softhand gripper

```bash
roslaunch jsk_baxter_startup baxter_softhand.launch
```

### CODE REVIEW

**Review by pazeshun — CHANGES_REQUESTED** [review]
Please update https://github.com/jsk-ros-pkg/jsk_robot/blob/master/jsk_baxter_robot/jsk_baxter_startup/README.md

**Comment by knorth55** [comment]
@pazeshun thank you for your review. I update the readme.
https://github.com/knorth55/jsk_robot/blob/baxter-softhand-launch/jsk_baxter_robot/jsk_baxter_startup/README.md

**Review by pazeshun — CHANGES_REQUESTED** [review]
Sorry, but could you add when & how to use `jsk_baxter_udev/*` and `jsk_baxter_machine/env.sh` to README?
Or updating Baxer Instruction slide in JSK GoogleDrive is OK.

File: jsk_baxter_robot/jsk_baxter_startup/baxter.launch [hunk 1]
```diff
@@ -70,8 +107,23 @@
     </group>
   </group>
 
+  <!-- launch defaulut gripper -->
   <node if="$(arg launch_gripper_action)" pkg="baxter_interface" name="baxter_gripper_action_server" type="gripper_action_server.py" args="" output="screen"/>
 
+  <!-- launch softhands -->
+  <group ns="lgripper" if="$(eval arg('left_gripper_type') in ['softhand', 'softhand-v2'])">
+    <include file="$(find softhand_ros)/launch/softhand_left.launch"
```
Comment (pazeshun) [inline jsk_baxter_robot/jsk_baxter_startup/baxter.launch:115]: Please add `softhand_ros` to package.xml

File: jsk_baxter_robot/jsk_baxter_startup/jsk_baxter_sensors/baxter_realsense_torso.launch [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  <arg name="depth_height" default="480" />
+
+  <arg name="camera_frame_id" default="$(arg realsense_camera_ns)_color_optical_frame" />
+  <arg name="manager" default="$(arg realsense_camera_ns)_camera_manager" />
+  <arg name="input_cloud" default="/$(arg realsense_camera_ns)/depth_registered/quarter/points" />
+  <arg name="input_image" default="/$(arg realsense_camera_ns)/depth_registered/quarter/image" />
+
+  <!-- interactive marker -->
+  <node name="transformable_interactive_server"
+        pkg="jsk_interactive_marker" type="transformable_server_sample"
```
Comment (pazeshun) [inline jsk_baxter_robot/jsk_baxter_startup/jsk_baxter_sensors/baxter_realsense_torso.launch:18]: Please add `jsk_interactive_marker` to package.xml

File: jsk_baxter_robot/jsk_baxter_startup/jsk_baxter_sensors/baxter_realsense_torso.launch [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    <arg name="publish_tf" value="$(arg publish_tf)" />
+    <arg name="color_width"  value="$(arg color_width)" />
+    <arg name="color_height" value="$(arg color_height)" />
+    <arg name="depth_width"  value="$(arg depth_width)" />
+    <arg name="depth_height" value="$(arg depth_height)" />
+  </include>
+
+  <node name="realsense_resize_points_publisher"
+        pkg="nodelet" type="nodelet"
+        args="load jsk_pcl/ResizePointsPublisher $(arg realsense_camera_ns)/$(arg manager)">
```
Comment (pazeshun) [inline jsk_baxter_robot/jsk_baxter_startup/jsk_baxter_sensors/baxter_realsense_torso.launch:52]: Please add `jsk_pcl_ros` to package.xml

File: jsk_baxter_robot/jsk_baxter_startup/jsk_baxter_sensors/baxter_spherical_kodak.launch [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<launch>
+  <arg name="spherical_camera_ns" default="kodak_head" />
+
+  <node name="kodak_transform_publisher" pkg="tf" type="static_transform_publisher"
```
Comment (pazeshun) [inline jsk_baxter_robot/jsk_baxter_startup/jsk_baxter_sensors/baxter_spherical_kodak.launch:4]: Please add `tf` to package.xml (although it is already required by `dynamic_tf_publisher`)

File: jsk_baxter_robot/jsk_baxter_startup/jsk_baxter_sensors/elp_usb.launch [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+  <!-- machine -->
+  <arg name="machine" default="localhost" />
+  <machine name="localhost" address="localhost" default="true" />
+  <arg name="load_machinepath" default="false" />
+  <arg name="machinepath" default="false" />
+  <include file="$(arg machinepath)" if="$(arg load_machinepath)" />
+
+  <group ns="$(arg camera_name)">
+    <node name="elp_camera_node" pkg="libuvc_camera" type="camera_node"
```
Comment (pazeshun) [inline jsk_baxter_robot/jsk_baxter_startup/jsk_baxter_sensors/elp_usb.launch:13]: Please add `libuvc_camera` to package.xml

File: jsk_baxter_robot/jsk_baxter_startup/jsk_baxter_sensors/kodak_pixpro.launch [hunk 6]
```diff
@@ -0,0 +1,16 @@
+<launch>
+  <arg name="camera_name" default="kodak_head" />
+  <include file="$(find video_stream_opencv)/launch/camera.launch" >
```
Comment (pazeshun) [inline jsk_baxter_robot/jsk_baxter_startup/jsk_baxter_sensors/kodak_pixpro.launch:3]: Please add `video_stream_opencv` to package.xml

File: jsk_baxter_robot/jsk_baxter_startup/jsk_baxter_sensors/realsense.launch [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+  <arg name="color_height" />
+  <arg name="depth_width" />
+  <arg name="depth_height" />
+
+  <group ns="$(arg realsense_camera_ns)">
+    <param name="rgb_camera/global_time_enabled" value="true"/>
+    <param name="l500_depth_sensor/global_time_enabled" value="true"/>
+  </group>
+
+  <include file="$(find realsense2_camera)/launch/rs_rgbd.launch">
```
Comment (pazeshun) [inline jsk_baxter_robot/jsk_baxter_startup/jsk_baxter_sensors/realsense.launch:15]: Please add `realsense2_camera` to package.xml

File: jsk_baxter_robot/jsk_baxter_startup/jsk_baxter_sensors/realsense.launch [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+    <arg name="manager" value="$(arg manager)" />
+    <arg name="publish_tf" value="$(arg publish_tf)" />
+    <arg name="color_width"  value="$(arg color_width)" />
+    <arg name="color_height" value="$(arg color_height)" />
+    <arg name="depth_width"  value="$(arg depth_width)" />
+    <arg name="depth_height" value="$(arg depth_height)" />
+  </include>
+
+  <!-- subscribing /$(arg realsense_camera_ns)/depth/image_rect_raw/compressedDepth causes serious frequency drop. -->
+  <node name="depth_compress_republish" pkg="image_transport"
```
Comment (pazeshun) [inline jsk_baxter_robot/jsk_baxter_startup/jsk_baxter_sensors/realsense.launch:26]: Please add `image_transport` to package.xml

File: jsk_baxter_robot/jsk_baxter_startup/jsk_baxter_sensors/respeaker.launch [hunk 9]
```diff
@@ -0,0 +1,3 @@
+<launch>
+  <node name="respeaker_node" pkg="respeaker_ros" type="respeaker_node.py" respawn="true"/>
```
Comment (pazeshun) [inline jsk_baxter_robot/jsk_baxter_startup/jsk_baxter_sensors/respeaker.launch:2]: Please add `respeaker_ros` to package.xml

**Review by pazeshun — CHANGES_REQUESTED** [review]

**Comment by knorth55** [comment]
Thanks, I updated the README and package.xml.

**Review by pazeshun — APPROVED** [review]

**Pull request merged**